src/config/index.js

```diff
@@ -93,11 +57,19 @@ class Config {
    * Save the config in memory to disk
    */
   save() {
+    // Delete unsaved scripts
+    fse.readdir(SCRIPTS_DIRECTORY_PATH).then(scripts => {
+      scripts
+        .filter(scriptName => !this.scriptNames.includes(path.basename(scriptName, '.yml')))
```

Comment:
Should do a check that the file is actually a `.yml` file? I'm guessing `path` has a way to get just the file type.